lib/src/main/java/com/diffplug/spotless/scala/ScalaFmtStep.java

```diff
@@ -39,23 +34,17 @@ public class ScalaFmtStep {
 	// prevent direct instantiation
 	private ScalaFmtStep() {}
 
-	private static final Pattern VERSION_PRE_2_0 = Pattern.compile("[10]\\.(\\d+)\\.\\d+");
-	private static final Pattern VERSION_PRE_3_0 = Pattern.compile("2\\.(\\d+)\\.\\d+");
-	private static final String DEFAULT_VERSION = "3.0.8";
+	private static final String DEFAULT_VERSION = "3.5.9";
 	static final String NAME = "scalafmt";
-	static final String MAVEN_COORDINATE_PRE_2_0 = "com.geirsson:scalafmt-core_2.11:";
-	static final String MAVEN_COORDINATE_PRE_3_0 = "org.scalameta:scalafmt-core_2.11:";
 	static final String MAVEN_COORDINATE = "org.scalameta:scalafmt-core_2.13:";
```

Comment:
Hmmm, on second thoughts for this change in general I think making the Scala version configurable (along with the scalafmt version) would probably solve the Scala conflict issue?

Theoretically this would allow you to specify the Scala version in, something akin to

```groovy
spotless {
  scala {
     scalaMajorVersion = '2.13'
  }
}
```

Rather than hardcoding the scala version in the `MAVEN_COORDINATE` as it is now.

Wdyt?

Reply:
So I ended up implementing this, I just haven't committed this yet (not sure if this should be done in a separate PR and/or also need to test that it works otherwise its pointless).

My current implementation would look like this

```gradle
spotless {
  scala {
    target 'streams/**/*.scala'
    scalafmt("$versions.scalafmt").configFile('checkstyle/.scalafmt.conf').scalaMajorVersion('2.13')
    licenseHeaderFile 'checkstyle/java.header', 'package'
  }
}
```

with `.scalaMajorVersion('2.13')` being entirely optional

Reply:
I like this approach, and I think it's good to do it inside this PR.